src/libraries/System.IO.FileSystem/tests/File/Open.cs

```diff
@@ -42,6 +42,24 @@ protected override FileStream CreateFileStream(string path, FileMode mode, FileA
         }
     }
 
+    public class File_Open_str_fm_fa_fs_buffer_fo : FileStream_ctor_str_fm_fa_fs_buffer_fo
```

Comment:
Is this test name correct?  I thought the pattern was based on the arguments being accepted by the method, but these methods are all path,options rather than path,FileMode,FileAccess,FileShare,bufferSize,FileOptions.

Same for the subsequent change.

Reply:
I have aligned these tests with the existing options-based test classes from `FileStream`.